plugin/index.ts

```diff
@@ -1036,6 +1038,28 @@ function processInlineStylesWarning(
   }
 }
 
+function injectVersion(
+  path: BabelCore.NodePath<BabelTypes.Program>,
+  state: ReanimatedPluginPass
+) {
+  const injectedName = '_REANIMATED_VERSION_PLUGIN';
```

Comment:
I'm thinking of renaming this to `_REANIMATED_VERSION_BABEL_PLUGIN` but I'm okay with the current name as well.